pkg/depository/create.go

```diff
@@ -87,12 +92,55 @@ func NewCreateDepositoryCmd() *cobra.Command {
 				fmt.Print(string(resp))
 				return nil
 			} else {
-				// Generate Account
-				privKey, addr := randAccountAndPrivateKey()
-				// Get Nonce
-				non := getNonce(host, addr.String())
+				// get wallet account dir
+				if walletDir == "" {
+					walletDir = common.WalletConfigDir
+				}
+				walletDir = strings.TrimSuffix(walletDir, "/")
+				_, err := os.Stat(walletDir)
+				if err != nil {
+					return err
+				}
+
+				// get accounts
+				dirEntries, err := os.ReadDir(walletDir)
+				if err != nil {
+					return err
+				}
+
+				// read account info
+				obj := common.WalletConfig{
```

Comment:
we need another flag `--account` to specify which account file under wallet dir should  be used to sign this message

Reply:
The `--account` parameter needs to be added to get specific account information instead of the first one.